public/client.js

```diff
@@ -1,9 +1,46 @@
-// $(function () {
-// 	$.get('/trucks', function (trucks) {
-// 		var truckList = [];
-// 		trucks.forEach(function (truck) {
-// 			truckList.push('<li><a href="/trucks/' + truck.name + '/">' + truck.name + '</a></li>')
-// 		});
-// 		$('.truck-list').append(truckList);
-// 	});
-// });
\ No newline at end of file
+$(function () {
+	// Create a function for the truck list so that it can be called
+	// in order to switch between trucks and types.
+	// Immediately invoked to make truck list the default view
+	var truckView = (function truckView() {
```

Comment:
Wow, how did I miss that?  I could have sworn it was working both ways when I was troubleshooting!  I removed the IIFE and just called it before the click event.  Thanks for pointing that out!